agents/ui.py

```diff
@@ -141,7 +141,8 @@ def __init__(self, spec):
 
         self.timer = QtCore.QTimer()
         self.timer.timeout.connect(lambda: self.redraw())
-        self.timer.start(1000 / 20)
+        self._ups = 20  # updates per second
```

Comment:
Hvorfor ikke bare kalde den self._updates_per_second i stedet for at det skal forklares i en kommentar?

Reply:
Det giver vel egentlig god mening.